src/applications/_mock-form-ae-design-patterns/App.jsx

```diff
@@ -9,14 +9,14 @@ import FormFooter from 'platform/forms/components/FormFooter';
 
 import { teardownProfileSession } from 'platform/user/profile/utilities';
 
-import fallbackFormConfig from '../config/fallbackForm';
-import greenFormConfig from '../config/prefill/taskGreen/form';
-import yellowFormConfig from '../config/prefill/taskYellow/form';
-import purpleFormConfig from '../config/prefill/taskPurple/form';
-import { useLocalStorage } from '../hooks/useLocalStorage';
-import { TaskTabs } from '../components/TaskTabs';
-import { Portal } from '../components/Portal';
-import { LOCATIONS_TO_REMOVE_FORM_HEADER } from '../utils/constants';
+import greenFormConfig from './patterns/pattern1/TaskGreen/config/form';
```

Comment:
do we want to use the real pattern names here? or is numbering ok?

Reply:
i'd go with keeping the numbers because it indicates a chronological order. Also, it might be easier for us to refer to certain patterns in the code with the numbers.

Reply:
yea considering that the pattern names have changed several times, I though a more generic name is preferred in the code.

That being said, the routes themselves as far as the url names can be whatever we want, so I'm certainly fine with making those more human readable as we move forward.

Reply:
sounds good to me!!